ecutenco/ecutenco.md

```diff
@@ -0,0 +1,21 @@
+# Evghen Cutenco 
+## Profile Pic: https://ca.slack-edge.com/T0338ASF4KF-U05FH4P855J-e84efa154b26-512
+## Call me by this name: Evghen 
+## Cohort: July 2023
+## Email ID: evghen.kutsenco@gmail.com 
+## LinkedIn: NONE
```

Comment:
I highly recommend one, if you explicitly do not want to create one then reply to this comment by saying, " I do not want to create my LinkedIn account"

Reply:
ok, I try to create, thank you :)